<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontSelection</class>
 <widget class="QDialog" name="FontSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="FontNameList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="FontStyleList">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>121</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="FontSizeList">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>111</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FontSelection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FontSelection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FontNameList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>FontSelection</receiver>
   <slot>fontnameSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fontnameSelected()</slot>
 </slots>
</ui>
